docs/main/Solutioning/backup-and-restore/kasten-br-fs.md

```diff
@@ -0,0 +1,338 @@
+
+As an example, we will be using `openebs-hostpath` storageclass as a global persistence storageclass for kasten installation. 
+
+1. Install Kasten.
+
+```
+helm install k10 kasten/k10 --namespace=kasten-io --set global.persistence.storageClass=openebs-hostpath
+```
+
+2. Once you have installed Kasten, verify that Kasten has installed correctly.
+
+**Command**
+
+```
+kubectl get po -n kasten-io
+```
+
+**Output**
+
+```
+NAME                                    READY   STATUS    RESTARTS   AGE
+aggregatedapis-svc-6cff958895-4kq8j     1/1     Running   0          4m11s
+auth-svc-7f48c794f-jmw4k                1/1     Running   0          4m10s
+catalog-svc-55798f8dc-m7nqm             2/2     Running   0          4m11s
+controllermanager-svc-85679687f-7c5t2   1/1     Running   0          4m11s
+crypto-svc-7f9bbbbccd-g4vf7             4/4     Running   0          4m11s
+dashboardbff-svc-77d8b59b4d-lxtrr       2/2     Running   0          4m11s
+executor-svc-598dd65578-cmvnt           1/1     Running   0          4m11s
+executor-svc-598dd65578-d8jnx           1/1     Running   0          4m11s
+executor-svc-598dd65578-p52h5           1/1     Running   0          4m11s
+frontend-svc-7c97bf4c7d-xmtb7           1/1     Running   0          4m11s
+gateway-68cdc7846-x9vw4                 1/1     Running   0          4m11s
+jobs-svc-7489b594c4-zkx4n               1/1     Running   0          4m11s
+k10-grafana-5fdccfbc5c-jtlms            1/1     Running   0          4m11s
+kanister-svc-9f47747f5-kt4r6            1/1     Running   0          4m10s
+logging-svc-6846c585d8-hqkt2            1/1     Running   0          4m11s
+metering-svc-64d847f4c7-chk9s           1/1     Running   0          4m11s
+prometheus-server-cbd4d5d8c-vql87       2/2     Running   0          4m11s
+state-svc-84c9bcc968-n2hg4              3/3     Running   0          4m11s
+```
+
+### Create VolumeSnapshotClass
+
+Whenever Kasten identifies volumes that were provisioned via a CSI driver, it will search for a VolumeSnapshotClass with Kasten annotation for the identified CSI driver. It will then utilize this to create snapshots.
+
+Create a `VolumeSnapshotClass` with the following yaml:
+
+```
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  annotations:
+    k10.kasten.io/is-snapshot-class: "true"
+  name: csi-mayastor-snapshotclass
+deletionPolicy: Delete
+driver: io.openebs.csi-mayastor
+```
+
+### Validate Dashboard Access
+
+Use the following `kubectl` command to forward a local port to Kasten ingress port or change the 'svc' type from **ClusterIP** to **NodePort** to establish a connection to it.
+
+:::note
+By default, the Kasten dashboard is not exposed externally.
+:::
+
+In this example we have changed the 'svc' type to **NodePort**:
+
+1. Forward a local port to the Kasten ingress port.
+
+```
+kubectl --namespace kasten-io port-forward service/gateway 8080:80
+```
+
+2. Change the 'svc' type as **NodePort**.
```

Comment:
@datacore-tilangovan do we need this even after port-forwarding?

Reply:
Not required, kasten document has some steps like how an user can achieve the dashboard UI, i have just followed that. user can either use port forwarding,nodePort or ingress

Reply:
That's what I meant - it should be 1 or 2, not 2 after 1.

Reply:
got it. @balaharish7 change this as user can follow either one of these steps.